recipe/meta.yaml

```
@@ -36,19 +37,21 @@ test:
   source_files:
     - src/test/unit
   requires:
-    - pytest
-    - pytest-cov
+    - ansible
     - mock
     - pip
+    - pytest
+    - pytest-cov
+    - pytest-timeout
   imports:
     - ansible_runner
   commands:
     - python -m pip check
     - ansible-runner --version
     - ansible-runner --help
     - cd src/test/unit
-    - pytest --cov ansible_runner -vv -k "not test_env_vars"  # [osx]
-    - pytest --cov ansible_runner -vv  # [linux]
+    - pytest --cov ansible_runner -vv -k "not (test_env_vars or containerization)"  # [osx]
+    - pytest --cov ansible_runner -vv -k "not containerization"  # [linux]
```

```suggestion
    - pytest --cov ansible_runner -vv -k "not (containerization or test_prepare_env_sshkey)"  # [linux]
```
I did a bit of digging, and this test hangs indefinitely when run on its own:
https://github.com/ansible/ansible-runner/blob/2.0.2/test/unit/config/test__base.py#L153-L161

With this change the recipe builds locally for me.

---

just normalized all the skips together, thanks!